### PR TITLE
LPS-130155 Do not release yet

### PR DESCRIPTION
**Pull request by izaera**
Hey @brianchandotcom, we forgot to set `Liferay-Releng-Bundle: false` in `app.bnd` (you added the `.lfrbuild-releng-ignore` file but master still has `Liferay-Releng-Bundle: true`).

### CODE REVIEW

**Comment by markocikos** [comment]
ci:reopen

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/affafdb1e8...2f3c0ff64a

File: modules/apps/frontend-view-state/app.bnd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Liferay-Releng-App-Description: %liferay-releng-app-description
 Liferay-Releng-App-Title: ${liferay.releng.app.title.prefix} Frontend View State
-Liferay-Releng-Bundle: true
+Liferay-Releng-Bundle: false
```
Comment (ling-alan-huang) [inline modules/apps/frontend-view-state/app.bnd:3]: @izaera SF enforce `Liferay-Releng-Bundle` to `true`.
@jpince can you help to check what SF should fix?

/cc @hhuijser

Comment (izaera) [reply modules/apps/frontend-view-state/app.bnd:3]: :scream: Sorry about that. I couldn't imagine that changing only that line could trigger that SF failure :-(

**Pull request closed without merging**